modules/core/include/visp3/core/vpTime.h

```diff
@@ -85,4 +87,24 @@ VISP_EXPORT int wait(double t0, double t);
 VISP_EXPORT void wait(double t);
 };
 
+class VISP_EXPORT vpChrono
```

Comment:
`vpChrono` is to use the C++11 [`steady_clock`](https://en.cppreference.com/w/cpp/chrono/steady_clock), that is a monotonic clock.

Also, documentation of [`gettimeofday`](http://man7.org/linux/man-pages/man2/gettimeofday.2.html):

> The time returned by gettimeofday() is affected by discontinuous jumps in the system time (e.g., if the system administrator manually changes the system time).  If you need a monotonically increasing clock, see clock_gettime(2).

Also, by default a call to `chrono.start` reset the chrono but with `chrono.start(false)` it is possible to add the duration.
Other than that, it is redundant with `vpTime::measureTimeMs()`.